sources/lib/Model/FlexibleEntity.php

```diff
@@ -99,6 +99,7 @@ public function __construct(array $values = null)
     {
         $this->container[$var] = $value;
         $this->touch();
+        $this->addModifiedColumn($var);
```

Comment:
It's possible but is it really good idea because it's a different functionnality ? ping @chanmix51